code/style.css

```diff
@@ -1,7 +1,339 @@
+* {
+  margin: 0;
+  font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+}
+/*The heading session*/
+.container {
+  display: flex;
+  justify-content: center;
+  width: 100%;
+  height: 100vh;
+}
+
+.bg-video {
+  position: absolute;
+  object-fit: cover;
+  max-width: 100%;
+  min-width: 100%;
+  z-index: 0;
+  min-height: 100%;
+  max-height: 100vh;
+}
+
+.button{ 
+  border: white;
+  background-color: #a8a377;
+  color: white;
+  font-weight: bold;
+  position: absolute;
+  border-radius: 5px;
+  z-index: 2;
+  top: 20px;
+  right: 10px;
+  font-size: 20px;
+}
+
+.button:hover {
+  border: solid #a8a377 1px;
+  color: #a8a377;
+  background-color: transparent;
+}
+
+.slogan {
+  position: absolute;
+  top: 20px;
+  left: 20px;
+  z-index: 1;
+  font-family: 'Syncopate', sans-serif;
+  width: 60px;
+  color: white;
+}
+
+.logo {
+  position: absolute;
+  z-index: 2;
+  margin-top: 35vh;
+  width: 170px;
+  height: auto;
+}
+
+
+/* Form and FAQ section*/
+.form-and-faq-container {
+  display: flex;
+  flex-direction: column;
+  background-color: teal;
+  margin-top: 0;
+}
+
+.form-header {
+  font-family: 'Syncopate', sans-serif;
+  font-size: 25px;
+}
+
+/*The Form*/
+.form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: lightblue;
+  border-radius: 5px;
+  height: 900px;
+  margin: 10px;
+  
+}
+.contact-details {
+  margin-bottom: 30px;
+}
+
+label { 
+  display: block;
+  margin: 15px
+}
+
+input {
+  display: block;
+  font-size: 20px;
+  border: none;
+  border-radius: 5px;
+  box-shadow: 0 2px 5px 0 rgba(56, 43, 82,0.5)
+}
+
+.guest-input {
+  display: block;
+  border: none;
+  border-radius: 5px;
+}
+
+select {
+  font-size: 20px;
+  margin: 5px;
+}
+
+input:focus {
+  background-color: #c5e4f0;
+}
+
+.form-header {
+  text-align: center;
+  margin: 60px auto 25px auto;
+  text-transform: uppercase;
+}
+
+.form-title {
+  text-align: center;  
+}
+
+.radio-buttons {
+  display: flex;
+  margin-bottom: 10px;
+}
+
+.radio[type=radio] {
+  box-shadow: none;
+  width: 100%;
+  height: 15px;
+}
+
+.checkbox {
+  display: flex;
+  font-size: 15px;
+  color: green;
+}
+
+.submit {
+  margin: 30px;
+  right: 20px;
+  padding: 15px 30px;
+  background-color: white;
+  font-size: 15px;
+  border-radius: 5px;
+  border: none;
+  box-shadow: 5px 5px 10px rgb(110, 119, 96);
+  cursor: pointer;
+}
+
+.submit:active {
+  background-color: #3e8e41;
+  box-shadow: 8px 8px 10px #666;
+  transform: translateY(4px);
+}
+
+/* gradient-color on the button changes when hover */
+
+.submit:hover{
+  background: linear-gradient(-45deg, #a8a377, white, #a8a377);
+  background-size: 400% 400%;
+  animation: gradient 15s ease infinite;
+}
+
+@keyframes gradient {
+  0% {
+      background-position: 0% 50%;
+  }
+  50% {
+      background-position: 100% 50%;
+  }
+  100% {
+      background-position: 0% 50%;
+  }
+}
```

Comment:
Oh wow! I have not noticed the animation on button at first, but cool!  My advise is to try to use some more visible colors next time or put animation on button without hovering so the user pays attention while doing the form :)